remoting/src/main/java/org/apache/rocketmq/remoting/protocol/body/BrokerReplicasInfo.java

```diff
@@ -115,29 +114,32 @@ public Long getMasterBrokerId() {
         }
 
         public boolean isExistInSync(String brokerName, Long brokerId, String brokerAddress) {
-            return this.getInSyncReplicas().contains(new ReplicaIdentity(brokerName, brokerId, brokerAddress));
+            return this.getInSyncReplicas().contains(new ReplicaIdentity(brokerName, brokerId, brokerAddress, true))
+                    || this.getInSyncReplicas().contains(new ReplicaIdentity(brokerName, brokerId, brokerAddress, false)) ;
```

Comment:
The `ReplicaIndentity`'s field `isAlive` will not participate in the method `List::contains`. How about adding a constucter method like `public ReplicaIndentity(String brokerName, Long brokerId, String brokerAdress)` and the filed `isAlive` with the default value `false` ~

Reply:
You are right. Thanks a lot~